modules/imgproc/src/smooth.cpp

```diff
@@ -2272,7 +2319,26 @@ static bool ipp_GaussianBlur(InputArray _src, OutputArray _dst, Size ksize,
         if(!ippBorder.m_borderType)
             return false;
 
-        CV_INSTRUMENT_FUN_IPP(::ipp::iwiFilterGaussian, &iwSrc, &iwDst, ksize.width, (float)sigma1, ippBorder);
+        // IW 2017u2 has bug which doesn't allow use of partial inMem with tiling
+        if((((ippBorder.m_borderFlags)&ippBorderInMem) && ((ippBorder.m_borderFlags)&ippBorderInMem) != ippBorderInMem)) {
+            return false;
```

Comment:
1) Should we fallback to single-threaded version?
```
CV_INSTRUMENT_FUN_IPP(::ipp::iwiFilterGaussian, &iwSrc, &iwDst, ksize.width, (float)sigma1, ippBorder);
```

2) Should we check version macro: `IW_VERSION_COMPLEX <= 20170002` (in case of fix in the next release)?

3) Should we add new macro for disabled IPP feature [here](https://github.com/opencv/opencv/blob/0d86681b88d4605b49ea06e0e6218c1bcc36925c/modules/core/include/opencv2/core/private.hpp#L189-L208)?
```
#define IPP_DISABLE_FILTERING_INMEM_PARTIAL 1 // IW 2017u2 has bug which doesn't allow use of partial inMem with tiling
```